RoddyCore/src/de/dkfz/roddy/config/validation/DefaultValidator.groovy

```diff
@@ -70,6 +70,9 @@ class DefaultValidator extends ConfigurationValueValidator {
      * API Level 3.4+
      */
     boolean checkDollarSignUsage(ConfigurationValue configurationValue) {

```

Comment:
My interpretation of "check" is positive/affirmative, i.e. if I check that something is used, it is present. The semantics of the method here are, however, negative -- true == **no** dollar sign.

Rename to e.g. "containsNoDollarSign" or so.